src/vm/dllimportcallback.cpp

```diff
@@ -1412,6 +1412,8 @@ VOID UMThunkMarshInfo::RunTimeInit()
         new (&sigInfo) PInvokeStaticSigInfo(pMD);
     else
         new (&sigInfo) PInvokeStaticSigInfo(GetSignature(), GetModule());
+    if (pStubMD == NULL)
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
     if (sigInfo.GetCallConv() == pmCallConvCdecl)
```

Comment:
Is this right? Shouldn't the adjustment be applied to just `m_cbRetPop`?

The comment for `m_cbActualArgSize` says that it is cache for `m_pSig.SizeOfFrameArgumentArray`. It won't be the case with this adjustment.